src/App.tsx

```diff
@@ -16,47 +19,103 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [goodsArr, setGoodsArr] = useState(goodsFromServer);
+  const [sortValue, setSortValue] = useState('');
+  const [reverseValue, setReverseValue] = useState(false);
+  const [resetValue, setResetValue] = useState(false);
```

Comment:
```suggestion
  const [sortValue, setSortValue] = useState(SORT_BY.none);
  const [isReversed, setIsReversed] = useState(false);
```
redundant to keep is state resetValue and goods.

change just sortValue and isReversed and based on it make function outside of component whid will accept goodsFromServer, sortValue, isReversed states and return sortedGoods.